README.md

```diff
@@ -169,148 +169,230 @@
+Configure your shell to initialize starship. Select yours from the list below:
 
-   ```powershell
-   Invoke-Expression (&starship init powershell)
-   ```
+<details>
+<summary>Bash</summary>
 
-   #### Ion
+Add the following to the end of `~/.bashrc`:
 
-   Add the following to the end of `~/.config/ion/initrc`:
+```sh
+eval "$(starship init bash)"
+```
 
-   ```sh
-   # ~/.config/ion/initrc
+</details>
 
-   eval $(starship init ion)
-   ```
+<details>
+<summary>Cmd</summary>
 
-   #### Elvish
+You need to use [Clink](https://chrisant996.github.io/clink/clink.html) (v1.2.30+) with Cmd.
+Create a file at this path `%LocalAppData%\clink\starship.lua` with the following contents:
 
-   **Warning** Only elvish v0.17 or higher is supported.
-   Add the following to the end of `~/.elvish/rc.elv`:
+```lua
+load(io.popen('starship init cmd'):read("*a"))()
+```
 
-   ```sh
-   # ~/.elvish/rc.elv
+</details>
 
-   eval (starship init elvish)
-   ```
+<details>
+<summary>Elvish</summary>
 
-   #### Tcsh
+Add the following to the end of `~/.elvish/rc.elv`:
 
-   Add the following to the end of `~/.tcshrc`:
+```sh
+eval (starship init elvish)
+```
 
-   ```sh
-   # ~/.tcshrc
+Note: Only Elvish v0.17+ is supported
 
-   eval `starship init tcsh`
-   ```
+</details>
 
-   #### Xonsh
+<details>
+<summary>Fish</summary>
 
-   Add the following to the end of `~/.xonshrc`:
+Add the following to the end of `~/.config/fish/config.fish`:
 
-   ```sh
-   # ~/.xonshrc
+```fish
+starship init fish | source
+```
 
-   execx($(starship init xonsh))
-   ```
+</details>
 
-   #### Cmd
+<details>
+<summary>Ion</summary>
 
-   You need to use [Clink](https://chrisant996.github.io/clink/clink.html) (v1.2.30+) with Cmd. Add the following to a file `starship.lua` and place this file in Clink scripts directory:
+Add the following to the end of `~/.config/ion/initrc`:
 
-   ```lua
-   -- starship.lua
+```sh
+eval $(starship init ion)
+```
 
-   load(io.popen('starship init cmd'):read("*a"))()
-   ```
+</details>
 
-   #### Nushell
+<details>
+<summary>Nushell</summary>
 
-   **Warning** This will change in the future. Only nu version v0.33 or higher is supported.
-   Add the following to your nu config file. You can check the location of this
-   file by running `config path` in nu.
+Add the following to the end of your Nushell configuration (find it by running `config path`):
 
-   ```toml
-   startup = [
-    "mkdir ~/.cache/starship",
-    "starship init nu | save ~/.cache/starship/init.nu",
-    "source ~/.cache/starship/init.nu"
-   ]
-   prompt = "starship_prompt"
-   ```
+```toml
+startup = [
+ "mkdir ~/.cache/starship",
+ "starship init nu | save ~/.cache/starship/init.nu",
+ "source ~/.cache/starship/init.nu"
+]
+prompt = "starship_prompt"
+```
+
+</details>
+
+<details>
+<summary>PowerShell</summary>
+
+Add the following to the end of your PowerShell configuration (find it by running `echo $profile`):
```

Comment:
```suggestion
Add the following to the end of your PowerShell configuration (find it by running `$PROFILE`):
```

Minor nit: `echo` command in PowerShell is implicit. Variables can be directly queried. See more - https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/write-output?view=powershell-7.2#description

Reply:
Ooh, thanks for the heads up, Rashil! 👍